src/Models/Request/Point.php

```diff
@@ -8,17 +8,21 @@
 
 namespace Qdrant\Models\Request;
 
+use Qdrant\Models\MultiVectorStruct;
+use Qdrant\Models\VectorStruct;
+use Qdrant\Models\VectorStructInterface;
+
 class Point implements RequestModel
 {
     protected string $id;
-    protected array $vector;
+    protected VectorStructInterface $vector;
 
     /**
      * @var array|null Payload values (optional)
      */
     protected ?array $payload = null;
 
-    public function __construct(string $id, array $vector, array $payload = null)
+    public function __construct(string $id, VectorStructInterface $vector, array $payload = null)
```

Comment:
This is a breaking change, but makes more sense to me.

Reply:
For backward compatibility, we could still accept an array, and use that to construct a nameless `VectorStruct` I suppose.

Reply:
In fact, from my perspective, we don't have so many users yet. So, for this reason, we can just change some stuff freely.